test/built-ins/FinalizationGroup/prototype/cleanupSome/this-does-not-have-internal-cells-throws.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-finalization-group.prototype.cleanupSome
+description: Throws a TypeError if this does not have a [[Cells]] internal slot
+info: |
+  FinalizationGroup.prototype.cleanupSome ( [ callback ] )
+
+  1. Let finalizationGroup be the this value.
+  2. If Type(finalizationGroup) is not Object, throw a TypeError exception.
+  3. If finalizationGroup does not have a [[Cells]] internal slot, throw a TypeError exception.
+  4. If callback is not undefined and IsCallable(callback) is false, throw a TypeError exception.
+  ...
+features: [FinalizationGroup, WeakRef]
+---*/
+
+assert.sameValue(typeof FinalizationGroup.prototype.cleanupSome, 'function');
```

Comment:
Needs `assert.sameValue(typeof WeakRef, 'function');`

Reply:
Not really, the idea here is to prevent false positives, and the line we have a [`new WeakRef`](https://github.com/tc39/test262/pull/2192/files/c6aadc7119f66e464a8dcb73d9ff7f15632b2a58#diff-62d997a70c72ad8453ed30888803bb6fR35) will throw an error that will fail the test anyway.